lazyflow/operators/ioOperators/opStreamingHdf5Reader.py

```diff
@@ -86,10 +86,17 @@ def setupOutputs(self):
         assert len(axistags) == len( dataset.shape ),\
             "Mismatch between shape {} and axisorder {}".format( dataset.shape, axisorder )
 
-        # Configure our slot meta-info
-        self.OutputImage.meta.dtype = dataset.dtype.type
-        self.OutputImage.meta.shape = dataset.shape
-        self.OutputImage.meta.axistags = axistags
+        if 'c' is not axisorder:
+            self.OutputImage.meta.dtype = dataset.dtype.type
+            self.OutputImage.meta.shape = (dataset.shape[0], dataset.shape[1], 1)
```

Comment:
Your code will only work with 2D data. To make it work with more dims, please use:

`(*dataset.shape, 1)`, to append a channel axis